Sources/PackageLoading/ManifestJSONParser.swift

```diff
@@ -10,16 +10,30 @@
 //
 //===----------------------------------------------------------------------===//
 
-import Basics
 @_implementationOnly import Foundation
 import PackageModel
-import TSCBasic
 
+import struct Basics.InternalError
+import struct TSCBasic.AbsolutePath
+import protocol TSCBasic.FileSystem
+import enum TSCBasic.PathValidationError
+import struct TSCBasic.RegEx
+import struct TSCBasic.RelativePath
+import struct TSCBasic.StringError
 import struct TSCUtility.Version
 
 enum ManifestJSONParser {
     private static let filePrefix = "file://"
```

Comment:
still used? I think it was for some business logic that maybe moved?

Reply:
Yep, it's used by `sanitizeDependencyLocation`. I think that's business logic we should move from here into `PackageModel` (like most of this file), but that should be a separate cleanup PR.